incubator/binding-openapi-asyncapi.spec/COPYRIGHT

```diff
@@ -0,0 +1,13 @@
+Copyright ${copyrightYears} Aklivity Inc.
```

Comment:
This is the wrong copyright, should be aklivity community license copyright instead.